etc/container.yaml

```diff
@@ -69,3 +69,7 @@ services:
 
     mandate_key_builder:
         class: byrokrat\giroapp\Builder\MandateKeyBuilder
+
+    db_settings_engine:
```

Comment:
Oops. `db_settings_engine` should go in `db.yaml`. `donor_builder` should go in `container.yaml`.